vpdq/benchmark/benchmark.py

```diff
@@ -0,0 +1,151 @@
+from numpy import hamming
+import vpdq
+import subprocess
+import os
+import sys
+import argparse
+from pathlib import Path
+from contextlib import contextmanager, nullcontext
+from enum import Enum
+import time
+
+VPDQ_MATCH_DIST = 31
+DIR = Path(__file__).parents[2]
+VIDEOS = DIR / "tmk/sample-videos"
+OUTPUT = "hashes"
+CPP_EXEC = Path(__file__).parents[1] / "cpp/build/vpdq-hash-video"
+
+
+@contextmanager
+def timer(context: str, print_on_enter: bool = False):
+    if print_on_enter:
+        print(f"{context}...")
+    start = time.perf_counter()
+    end = start
+    yield lambda: end - start
+    end = time.perf_counter()
+    print(f"{context}: {end - start:.4f}s")
+
+
+def get_argparse() -> argparse.ArgumentParser:
+    ap = argparse.ArgumentParser(description=__doc__)
```

Comment:
nit: setting =__doc__ doesn't do anything, it's an empty string here. Either add a doc or remove the description